qml/singletons/GlobalConfig.qml

```diff
@@ -41,6 +41,8 @@ Item {
 
 
 
+    property real loggingDuration : 0;
+    property bool scheduledLogging : false;
 
```

Comment:
Now we have them in GlobalConfig we can make them persistent - what do you think?